doc/api/worker.md

```diff
@@ -0,0 +1,466 @@
+# Worker
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+The `worker` module provides a way to create multiple environments running
+on independent threads, and to create message channels between them. It
+can be accessed using:
+
+```js
+const worker = require('worker');
```

Comment:
If this is exposed without a flag, this is a `semver-major` change as I presume.
/cc @jasnell

Reply:
As @benjamingr said below, this module is essentially not in public use, and the creation timestamp is, as I understand it, a bit misleading because the package name was only recently taken over by another author.